desktop/cmp/rest/RestGridModel.js

```diff
@@ -95,6 +106,7 @@ export class RestGridModel {
      * @param {Object[]|RecordAction[]} [menuActions] - actions to display in the grid context menu. Defaults to add, edit, delete.
      * @param {Object[]|RecordAction[]} [formActions] - actions to display in the form toolbar. Defaults to delete.
      * @param {Object} [actionWarning] - map of action (e.g. 'add'/'edit'/'delete') to string.  See default prop.
+     * @param {function} [prepareCloneFn] - a function used by the clone action to prepare a clone of a record.
```

Comment:
Please look at documenting the signature here via an `@callback` typedef - e.g. https://github.com/exhi/hoist-react/blob/395e285e9281373aad81574e1d6dca5c302fa24b/data/RecordAction.js#L146

Reply:
Done. Added following documentation, and renamed callback function to `prepareCloneCb`.

`    /**
     * @callback prepareCloneCb
     * @param {record} original record
     * @param {clone} record to be cloned
     */
`